bot/cogs/watchchannels/watchchannel.py

```diff
@@ -285,7 +285,10 @@ async def send_header(self, msg: Message) -> None:
         else:
             message_jump = f"in [#{msg.channel.name}]({msg.jump_url})"
 
-        footer = f"Added {time_delta} by {actor} | Reason: {reason}"
+        # Add reason to the footer if it exists.
+        footer = f"Added {time_delta} by {actor}"
+        if reason:
+            footer += f" | Reason: {reason}"
```

Comment:
When I come across situations like this, I realized it's usually a little bit easier to reason about the formatting when you include the stem in the extension f-string like so:
```python
footer = f"{footer} | Reason: {reason}"
```
This is not even part of the review though, what you have is entirely fine.